src/main/sensors/gyro_filter_impl.h

```diff
@@ -0,0 +1,37 @@
+static FAST_CODE void GYRO_FILTER_FUNCTION_NAME(gyroSensor_t *gyroSensor, timeDelta_t sampleDeltaUs)
```

Comment:
This file serves as a makeshift template to generate a functions based on policy: with debug setting of variables, or without it.
As a result it is included twice with different set of `GYRO_FILTER_FUNCTION_NAME` and `GYRO_FILTER_DEBUG_SET` macro definitions and for it to work it lacks `#pragma once`.

I could probably get away with something like this, but there are not a lot of reason to do so:
```c
#ifndef GYRO_FILTER_##GYRO_FILTER_FUNCTION_NAME##_H
#define GYRO_FILTER_##GYRO_FILTER_FUNCTION_NAME##_H
// ...
#endif
```

In C++ we could replace this function by a template:
```cpp
template<Policy debugSet>
void filterGyro(...) {
    debugSet(...);
}
```

And then use as follows:
```cpp
if (gyroDebug != DEBUG_NONE) {
    filterGyro<EmptyFunction>(...);
} else {
    filterGyro<DebugSet>(...);
}
```